dashboard/src/components/Catalog/CatalogItems.tsx

```diff
@@ -28,25 +30,16 @@ export default function CatalogItems({
     () =>
       availablePackageSummaries.map(c => {
         return {
+          // TODO: this should be simplified once the operators are also implemented as a plugin
           type: `${c.availablePackageRef?.plugin?.name}/${c.availablePackageRef?.plugin?.version}`,
           id: `package/${c.availablePackageRef?.identifier}`,
           item: {
-            plugin: c.availablePackageRef?.plugin ?? ({ name: "", version: "" } as Plugin),
-            id: `package/${c.availablePackageRef?.identifier}/${c.latestVersion?.pkgVersion}`,
             name: c.displayName,
-            icon: c.iconUrl ?? placeholder,
-            version: c.latestVersion?.pkgVersion ?? "",
-            description: c.shortDescription,
-            // TODO(agamez): get the repo name once available
-            // https://github.com/kubeapps/kubeapps/issues/3165#issuecomment-884574732
-            repo: {
-              name: c.availablePackageRef?.identifier.split("/")[0],
-              namespace: c.availablePackageRef?.context?.namespace,
-            } as IRepo,
```

Comment:
It's not clear to me from context how all this data can be removed (or replaced with availablePackageSummary) - unless you've updated elsewhere to populate that.

Reply:
I didn't have the chance to write some comments to ease the review, but, yep, of course, I had to update it elsewhere. Particulary, this info is being used in the `<InfoCard>` that is rendered at `PackageCatalogItem.tsx`.

Ideally, these components would get simplified to just an `AvailablePackageSummaryItem`, but currently we sill to differentiate between a package and an operator.
Wherefore I just simply extracted the operator things (namespace, description, etc) back to the `OperatorCatalogItem` and simplify as much as I could the package item. However, I had to leave some fields (like the `name: c.displayName` as they were being used in the `CatalogItem`.

But, at least, we are converging, step by step, in simplifying the UI logic in favor of the new package data model.

Reply:
Excellent. Thanks for the explanation :)